### DISCUSSION TITLE
２回目のファー厶ウェア書き込みが出来ない

### DISCUSSION THREAD

**Discussion by hisui-studio** [Yowkees/keyball#259] [Q&A] [answered]
REMAPにてKeyball61_testの書き込みに成功したのですが、2度目の書き込みができません。

他のQ&Aを見たところ2度目以降FLASH押下後にリセットスイッチを2回押下するとできるとの記載があったのですがファームウェア書き込みの候補に検出されません。QMKも同様です。

確認したこと
•リセットボタンを押す度に点滅しているのでリセットボタンの接続はできているようです。

•USB接続した際PC側にもデバイス検出はされているのでPCとの接続はできています。

•リセットボタン押下のタイミングや回数を複数パターン試しましたが、検出されませんでした。

•左右両方とも同じ症状でProMicroを交換しても同様でした。

解決方法をご教示頂けますと幸いです。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

ProMicroをKeyball基板から取り外して、ProMicro単体で書き込んでみてください。
下記リンクの画像のようにピンセットでRSTとGNDをショートさせると（リセットSWを押したのと同じ動作）書き込みモードに入り、認識されると思います。
https://kbdbuild.vercel.app/blog/buildguide_yamada_promicro

ProMicro単体でも認識されない場合、ProMicroの初期不良の可能性もありますので、ProMicroのご購入先へお問合せいただければ幸いです。

以上、お手数ですがご対応お願いします。
